data/journal_check/bug_refs.json

```
@@ -559,6 +559,13 @@
         },
         "type": "bug"
     },
+    "bsc#1229146": {
+        "description": "Job cockpit-ws-user.service/stop deleted to break ordering cycle starting with basic.target/stop",
```

Regex needs escaping:

```suggestion
        "description": "Job cockpit-ws-user\\.service/stop deleted to break ordering cycle starting with basic\\.target/stop",
```